src/ext/context.rs

```diff
@@ -31,7 +31,7 @@ impl ContextExt for dc::Context {
                 let base_alias = file_stem.to_str_or_err()?.to_owned();
 
                 // Get the branch.  If available, this will be stored in the query.
-                match url.fragment() {
+                match git_url.branch() {
```

Comment:
The fragment might be something like `#branch:folder`, but we don't want the folder in the alias.

Reply:
That's a useful rule of thumb 👍